app.rb

```diff
@@ -0,0 +1,103 @@
+require './classes/person'
+require './classes/student'
+require './classes/classroom'
+require './classes/teacher'
+require './classes/capitalize_decorator'
+require './classes/trimmer_decorator'
+require './classes/book'
+require './classes/rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def options
+    puts 'Please enter the number of the option:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def list_books
+    if @books.empty?
+      puts 'There are no books available'
+    else
+      puts 'This is the list of books available:'
+      @books.each do |book|
+        puts "- #{book.title} by #{book.author}"
+      end
+    end
```

Comment:
- [ ] Great job so far implementing the method for listing books. However, I would like to recommend that you list the books with their `index` such that when users are creating a rental they can select the specific book they are creating the rental for. Currently, because you don't have that am unable to create a rental because I don't know how to select the user I want to create a rental for.

  - Kindly consider adding the `key` of the `data` you're displaying. When displaying the list of books you should display it with the `title` and `author` keys.
  
<table>
<tr>
<th align=“middle”>Main design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="667" alt="Screenshot 2023-09-02 at 23 07 50" src="https://github.com/sergio-usma/ruby_school_library/assets/107192076/92fe442d-c541-44a8-9660-2dea6e608dd4">
</td>
<td><img width="407" alt="Screenshot 2023-09-02 at 23 07 36" src="https://github.com/sergio-usma/ruby_school_library/assets/107192076/f3310906-26a6-4c71-8f33-18b3c8f87efd"></td>
</tr>
</table>